pyaerocom/scripts/cams2_83/engine.py

```diff
@@ -8,11 +8,20 @@
 from reprlib import repr
 
 import numpy as np
+import pandas as pd
 import xarray as xr
+from tqdm import tqdm
```

Comment:
tqdm not used. How did this file get through ruff/pre-commit? pre-commit not set up?

Reply:
🤷 no idea, it is set up